mautic/mautic
# Dpmma 1743 focus actions on contact timeline

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ no]
| New feature/enhancement? (use the a.x branch)      | [ yes]
| Deprecations?                          | [ no]
| BC breaks? (use the c.x branch)        | [no ]
| Automated tests included?              | [ yes] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR adds Focus Item events like view, click in Contacts timeliene.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create Focus Item with trackable link
3. Identify contact 
4. Go to page with working Focus Item
5. Check contact timeline
6. Click link from Focus Item
7. Check contact timeline


![image](https://user-images.githubusercontent.com/100592709/220212174-a30bd9b9-5a3b-47f7-bc16-d81f6cb9f344.png)
